sdk/python/kfp/_client.py

```diff
@@ -338,8 +340,69 @@ def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, para
         pipeline_id=pipeline_id,
         workflow_manifest=pipeline_json_string,
         parameters=api_params)
+    return JobConfig(spec=spec, resource_references=resource_references)      
+
+  def run_pipeline_recurring(self, experiment_id, job_name, start_time=None, end_time=None, interval_second=600 , max_concurrency=1, params={}, pipeline_package_path=None, pipeline_id=None, namespace=None, enabled=True):
+     """Create a recurring run.
+    Args:
+      experiment_id: The string id of an experiment.
+      job_name: name of the job.
+      start_time: The time string of the time when to start the job.
+      end_time: The time string of the time when to end the job.
```

Comment:
actually can you change this to be the `RFC3339` string?

Reply:
you mean in the comment?

Reply:
yeah